Kernel/Problem.hpp

```diff
@@ -191,7 +191,7 @@ class Problem {
   UnitList* _units;
   DHMap<unsigned,Literal*> _deletedFunctions;
   DHMap<unsigned,Unit*> _deletedPredicates;
-  DHMap<unsigned,Unit*> _partiallyDeletedPredicates; 
+  DHMap<unsigned,Unit*> _partiallyDeletedPredicates;
```

Comment:
Since I am reading this anyway and I suspect you understand this better...when is a predicate only partially deleted? Could there be a comment somewhere?

Reply:
It's when UPDR decides only one impliction of an <=> is needed. 

Note that this will all get revised in a subsequent PR. (If you remember the big `#if 0` section we put into FiniteModelBuilder before CASC, I made it even bigger here, so these (partially)deleted creatures currently don't play any role at all.)